projects/solidity/build.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash -eu
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Build boost statically, linking it against libc++
+cd $SRC/boost
+./bootstrap.sh --with-toolset=clang
+./b2 clean
```

Comment:
Does this use the environment's `CFLAGS`?
Otherwise this will break with MSAN.

Reply:
It picked up CFLAGS for ASan, so I'm guessing it respects `CFLAGS` but I'm not sure.